### ISSUE TITLE
ScrollBar control does not support UIA accessibility 

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4983]
**Environment**
- Microsoft Windows [Version 10.0.19041.388]
- .NET Core SDK 6.0.100-preview.3.21202.5

**Have you experienced this same bug with .NET Framework?:**
- Yes

**Problem description**
![image](https://user-images.githubusercontent.com/23376742/119790447-9e215800-bedc-11eb-918f-7b58467e6397.png)

Accessibility providers:
`[pid:10384,providerId:0x811196 Main:Nested [pid:10832,providerId:0x811196 Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)]; Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]`

**Expected behavior:**
There should be UIA accessibility provider from WinForms.

**Minimal repro:**
Create WinForms app with TrackBar control and analyze it using Inspect.

**Related item:**
This work item is the sub of #3037

**Comment by Marie-Shi** [comment]
Verified this issue with .NET dotnet-sdk-6.0.0-rc.1.21368.1 from latest main branch, it is fixed, now ScrollBar control supports UIA accessibility.
![UIA](https://user-images.githubusercontent.com/45864017/126128421-bb4ed198-b2d6-462e-a5bb-a7b1ae4af016.png)

**Comment by Lydia-Shi** [comment]
Verified with .NET 6.0.100-preview.7.21369.19 build, this issue has been fixed that ScrollBar control supports UIA accessibility.
